### DISCUSSION TITLE
Need Clarification on the Doubts whether it work or not on VSK Dashboard

### DISCUSSION THREAD

**Discussion by vskmizoram** [Sunbird-cQube/community#858] [Q&A]
I have installed one server and one client desktop computer with a private IP (192.168.1.9 (Server) and 192.168.1.10 (Desktop) on my premises and installed successfully the VSK Dashboard on the Server as localhost on IP 192.168.1.9

The client machine is successfully communicating with the Server using private IP and we are getting the VSK Dashboard with all 6 applications of VSK on both the machine (i.e client and server) using IPs 192.168.1.9:4200 and 192.168.1.10:4200

Now I have the Static IP provided by ISP (i.e 59.93.129.... ), I need clarification on :

1. If I get the hardware firewall and map the Static IP Provided by ISP and redirect to 192.168.1.9:4200 (through port and IP forwarding), will  I get the VSK Dashboard with all 6 applications using the Static IP 59.93.129..... 

2. If Yes, then If I link this static IP with my domain vskmizoram.com , will I be able to access the VSK Dashboard with all 6 applications using the vskmizoram.com

3. If No, then what process is to be adopted to access the VSK Dashboard from the external network with the public static IP?

**Comment by vskmizoram** [comment]
@Rajesh... Thanx for the suggestion and guidance... I will try that and get back to you

**Reply by bommalla** [reply]
Sure..Thanks
